cou/apps/auxiliary.py

```diff
@@ -93,3 +92,27 @@ def channel_codename(self) -> OpenStackRelease:
                 f"for channel: '{self.channel}'"
             )
         )
+
+
+@AppFactory.register_application(["rabbitmq-server"])
+class RabbitMQServer(OpenStackAuxiliaryApplication):
+    """RabbitMQ application.
+
+    RabbitMQ had to wait until the entire model will be idle after the upgrade to ensure other
+    upgrade can continue.
+    """
+
+    wait_timeout = 300
+    wait_for_model = True
+
+
+@AppFactory.register_application(["keystone"])
+class Keystone(OpenStackApplication):
```

Comment:
Please move `Keystone` to `app.py`. This is not an auxiliary charm

Reply:
after #115 is merged.

Reply:
I must move core.py to base.py to be able to move Keystone to coro.py, otherwise there will be circular import.